app/assets/src/components/common/CoverageVizBottomSidebar/utils.js

```diff
@@ -15,7 +15,7 @@ export const getHistogramTooltipData = memoize(
         data: [
           [
             "Base Pair Range",
-            `${Math.round(coverageObj[0] * binSize)} - ${Math.round(
+            `${Math.round(coverageObj[0] * binSize)}\u2013${Math.round(
```

Comment:
are we not allowing full utf-8 literals in our codebase?

Reply:
I decided to keep it in the character encoding because the en-dash looks a lot like a hyphen, and I don't want someone to accidentally change it to a hyphen in the future. Added a comment that "\u2013 is en-dash". We can change this approach if we start using a lot of these unicode characters.